picard/util/webbrowser2.py

```diff
@@ -26,10 +26,8 @@
 from picard.const import PICARD_URLS
 
 
-"""
-A webbrowser extension for Picard. It handles and displays errors
-in PyQt and also adds a utility function for opening Picard URLS.
-"""
+# A webbrowser extension for Picard. It handles and displays errors
```

Comment:
This is no longer a valid docstring?

Reply:
In fact, it never was, since it's not at the start of file.
I'll convert it in a valid docstring, but since we only have few in code, module docstring or comment is the same.

Reply:
Fixed in https://github.com/metabrainz/picard/pull/1146/commits/8fe3ee23d82944ad244bfc8941c277eb3c93fb90